src/main/java/org/mdbenefits/app/preparers/ApplicantNeedsPreparer.java

```diff
@@ -0,0 +1,38 @@
+package org.mdbenefits.app.preparers;
+
+import formflow.library.data.Submission;
+import formflow.library.pdf.PdfMap;
+import formflow.library.pdf.SingleField;
+import formflow.library.pdf.SubmissionField;
+import formflow.library.pdf.SubmissionFieldPreparer;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+@Component
+public class ApplicantNeedsPreparer implements SubmissionFieldPreparer {
+
+        @Override
+        public Map<String, SubmissionField> prepareSubmissionFields(Submission submission, PdfMap pdfMap) {
+                Map<String, SubmissionField> results = new HashMap<>();
+
+                Map<String, Object> inputData = submission.getInputData();
+
+                ArrayList<String> programs = (ArrayList) inputData.get("programs[]");
```

Comment:
For the declaration part, you could just use List, as it's the parent type for ArrayList(). 
Not really necessary, but kinda handy - like the way on line 19 where you just declare "Map<String, SubmissionField>", but it's really a HashMap. 

`List<String> programs = (List) inputData.get("programs[]");` should work.  We don't actually know that it's an ArrayList, and really we don't need to care about that - we only care that it behaves like a list.

Reply:
I did not know that List is the parent type of arrayList. Thank you for sharing! I'll make these changes